oiosaml/src/test/java/dk/gov/oio/saml/servlet/LogoutRequestHandlerTest.java

```diff
@@ -116,17 +113,39 @@ public void testIdPLogoutRequestWhenLoggedIn() throws Exception {
 		Mockito.when(request.getParameter("SAMLRequest")).thenReturn(base64EncodedMessage);
 
 		// Mock DummyOutputStream
-		DummyOutputStream dummyOutputStream = Mockito.mock(DummyOutputStream.class);
+		DummyOutputStream dummyOutputStream = Mockito.spy(new DummyOutputStream());
```

Comment:
@sihpe or @jpramming do you have an opinion on mock vs spy?

Reply:
Spy is used to enabling validation of content on the output stream

Reply:
There is no longer a reason to validate generated HTML so cleaned up the code